website/docs/guides/pandas_integration.md

```diff
@@ -69,3 +69,38 @@ with HyperProcess(Telemetry.DO_NOT_SEND_USAGE_DATA_TO_TABLEAU) as hyper:
 pantab currently does not yet support reading `NUMERIC` types.
 The query shown above works around this issue by using a `CAST`.
 :::
+
+
+## Communicate via Arrow files
+
+You can use the [Apache Arrow format](https://arrow.apache.org/) to communicate between Hyper and Arrow.
```

Comment:
I think "communicate" only makes sense you are talking about two separate systems or process, not one system and a format. Maybe like this:

```suggestion
## Importing and exporting Arrow data with Pandas

You can use the [Apache Arrow format](https://arrow.apache.org/) to import or export data between Hyper and Pandas.
```